.github/workflows/ci.yaml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [18.12.0, 20.9.0]
+        node: [18.20.5, 20.18.1, 22.12.0, 23.3.0]
```

Awesome work!! It would be better to have a 
canary in the coal mine to spot future breakage.
```suggestion
        node: [18.20.5, 20.18.1, 22.12.0, latest]
```

---

Hmm only when we push though. I wish I'd pinned more versions in the prebuild scripts because it contributed to random failures.

I think new Node versions are something we have to explicitly do work for rather than something that should start potentially breaking builds. @LinusU any opinions?

Sorry about your other PR, I looked for existing ones but somehow still missed it.

---

My sense with `latest` is that there is a chance that CI tells us we have a problem before end users complain and that the non-latest jobs assure us an only the bleed-edge end users are seeing those problems.